index.html

```diff
@@ -3001,6 +2999,38 @@ <h3>
 
         </section>
 
+        <section class="normative">
+            <h3>
+                Add Index
```

Comment:
```suggestion
                Adding an index
```

W3C style guides are moving to "sentence capitalization for titles" based on some longitudinal study they did on titles. Don't know if this title fits, but we might think about how folks would read this entry in a table of contents. The section seems to be about "adding an index"?

Reply:
Done!